_posts/events/2014-04-26-mwe.md

```diff
@@ -0,0 +1,82 @@
+ * Lexicon-grammar interface: representing, at the lexical level, phenomena such as agreement, discontinuity and free word order; construction of MWE lexicons which would be easily convertible and maximally reusable in different parsing frameworks.
+ * “Deep” parsing techniques for MWEs: optimal representation of MWEs within linguistic frameworks, such CCG, HPSG, LFG, TAG, minimalism, etc; processing MWEs before, during or after parsing; representing the semantics of MWEs.
+ * Hybrid parsing of MWEs: combining data-driven and knowledge-based approaches for efficient and linguistically precise parsers; using unannotated data for improving models based on annotated data.
+ * Annotating MWEs in treebanks: MWE-aware methodologies of treebank construction, and their increased usability in parsing.
+
+This special track is endorsed by PARSEME, European IC1207 COST Action, dedicated to parsing and MWEs. A separate time slot will be allocated to the special track within the workshop. Authors may submit papers either to the special track or to the regular workshop. They should follow common submission instructions.
+
+PARSEME TRAVEL GRANTS
+
+PARSEME will fund travel and stay for at least 20 participants from the Action's member countries. The deadline for grant applications is 3 March 2014. Click here for more information on the application procedure.
+
+SUBMISSION MODALITIES
+For MWE 2014, we accept the following two types of submissions:
+ * Regular long papers (8 content pages + 1 page for references): Long papers should report on solid and finished research including new experimental results, resources and/or techniques.
+ * Regular short papers (4 content pages + 1 page for references): Short papers should report on small experiments, focused contributions, ongoing research, negative results and/or philosophical discussion.
+
+The reported research should be substantially original. The papers will be presented orally or as posters. The decision as to which papers will be presented orally and which as posters will be made by the program committee based on the nature rather than on the quality of the work. All submissions must be in PDF format and must follow the EACL 2014 formatting requirements (download ZIP archive). We strongly advise the use of the provided Word or LaTeX template files.
+
+Reviewing will be double-blind, and thus no author information should be included in the papers; self-reference should be avoided as well.
+
+Resources submitted with the papers should be anonymized for submission. Papers and/or resources that do not conform to these requirements will be rejected without review. Accepted papers will appear in the workshop proceedings, where no distinction will be made between papers presented orally or as posters.
+
+Submission and reviewing will be electronic, managed by the START system:
+
+https://www.softconf.com/eacl2014/MWE/
```

Comment:
Put `<` and `>` around the URL so that it is rendered as a link.

Reply:
Done